packages/atlas/src/components/_buttons/Button/Button.tsx

```diff
@@ -64,10 +66,11 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>((props, ref) =>
       disabled={disabled}
       aria-disabled={disabled}
       variant={variant}
-      iconOnly={!!iconOnly}
+      iconOnly={iconOnly}
```

Comment:
Missed that before, but it looks like iconOnly button variant is oversized: 
large: 56px vs 48
medium: 46px vs 40 
small: 34px vs 32